tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -333,14 +342,17 @@ CalculationFunc calculation_func(CaseParam const& param) {
     };
 
     if (calculation_type == "power_flow"s) {
-        return [sym, get_default_options](MainModel& model, CalculationMethod calculation_method,
-                                          Dataset const& dataset, ConstDataset const& update_dataset, Idx threading) {
-            if (sym) {
-                return model.calculate_power_flow<symmetric_t>(get_default_options(calculation_method, threading),
-                                                               dataset, update_dataset);
+        return [param, get_default_options](MainModel& model, CalculationMethod calculation_method,
+                                            Dataset const& dataset, ConstDataset const& update_dataset, Idx threading) {
+            auto options = get_default_options(calculation_method, threading);
+            options.optimizer_type = param.tap_changing_strategy == "disabled"
+                                         ? OptimizerType::no_optimization
+                                         : OptimizerType::automatic_tap_adjustment;
+            options.optimizer_strategy = optimizer_strategy_mapping.at(param.tap_changing_strategy);
```

Comment:
there's probably a better way of doing things like this but since we are going to touch this when we pull calculation type + symmetry into the options as well, i don't think it's necessary to make this nicer than it is now.